### DISCUSSION TITLE
Hover with peer-checked

### DISCUSSION THREAD

**Discussion by hrishikesh-k** [tailwindlabs/tailwindcss#5498] [Help] [answered]
Hi folks,

Tailwind CSS is purely amazing. Thank you to all the collaborators for their hardwork.

I do have a question though, wondering if someone can point me in the right direction. I'm using `jit` mode and the `peer` selector is working correctly. However, I wonder if it's possible to apply the hover state to `peer-checked`? I currently have the following:

```html
<input class = "hidden peer" name = "name" type = "radio" value = "value"/>
<span class = "text-white bg-grayDarker hover:bg-grayDarkest peer-checked:font-medium peer-checked:text-tealDarkest peer-checked:bg-tealLighter hover-peer-checked:bg-tealLightest">Text</span>
```

Out of all the `peer` classes only the last one doesn't work. I've also tried `peer-checked-hover`. I'm not really sure what the order should be in this case. Do I need to enable it from the config or something like that?

EDIT: Note that, I want to trigger the background colour change when I hover on `span` and not `input`. So, `peer-hover` might not work.

**Comment by horita-yuya** [comment, marked as answer]
Tailwind jit mode supports a feature combining variants.

`peer-checked:hover:bg-tealLightest` would work.

**Reply by hrishikesh-k** [reply]
That was it, thank you very much!
